include/bluetooth/services/latency.rst

```diff
@@ -0,0 +1,38 @@
+.. _latency_readme:
+
+GATT Latency Service
+####################
+
+The GATT Latency Service is a custom service that receives data and returns result about GATT latency operations.
+
+To measure the GATT transmission latency, when a client writes a timestamp to the latency characteristic on the server, the server will automatically reply back.
+The client can then use its present time and the corresponding timestamp written before to estimate the round-trip time (RTT).
+
+The GATT Latency Service is used in the :ref:`ble_llpm` sample.
+
+Service UUID
+************
+
+The 128-bit service UUID is 67136E01-58DB-F39B-3446-FDDE58C0813A.
+
+Characteristics
+***************
+
+This service has one characteristic.
+
+Latency Characteristic (67136E02-58DB-F39B-3446-FDDE58C0813A)
+=============================================================
+
+Write
+   Write data to the characteristic and wait for a response to compute the time spent.
+
+
+API documentation
+*****************
+
+| Header file: :file:`include/latency.h`
```

Comment:
@greg-fer , does this work? I would expect we should use `include/bluetooth/services/latency.h`

Reply:
You're right, the path needs to be updated.